packages/console/src/containers/ErrorBoundary/index.tsx

```diff
@@ -11,6 +11,23 @@ import AppError from '@/components/AppError';
 import SessionExpired from '@/components/SessionExpired';
 import { isInCallback } from '@/utils/url';
 
+/**
+ * Returns true if the error is an OIDC invalid_grant error.
+ *
+ * OIDC request error is globally converted to the LogtoRequestError.
+ * @see {@link @logto/core/packages/core/src/middleware/koa-oidc-error-handler.ts}
+ * We need to check the error code to determine if it is an OIDC invalid_grant error.
+ */
+const isOidcInvalidGrantError = (error: Error) => {
+  const oidcGrantErrors = ['oidc.invalid_grant', 'oidc.invalid_target'];
```

Comment:
nit: This can be declared after the if-condition check

Reply:
feels more like a util method. If put in the component, it will be triggered on every cycle render (of course if pass the above condition assertions)

Reply:
Oh, I mean
```ts
if (!(error instanceof LogtoRequestError)) {
  return false;
}

const oidcGrantErrors = ['oidc.invalid_grant', 'oidc.invalid_target'];
return oidcGrantErrors.includes(error.code);
```

Reply:
k updated